GDJS/Runtime/runtimegame.ts

```diff
@@ -913,19 +913,34 @@ namespace gdjs {
     /**
      * Helper function to get information about the platform running the game.
      */
-    getPlatformInfo = () => ({
-      // @ts-ignore
-      isCordova: !!window.cordova,
-      devicePlatform:
+    getPlatformInfo = () => {
+      const supportedCompressionMethods: ('cs:gzip' | 'cs:deflate')[] = [];
```

Comment:
nit: this could be saved in a variable (outside of the class) so that it's computed once (`if (!supportedCompressionMethods) { supportedCompressionMethods = []; ... }`)